reprohack_hub/forms.py

```diff
@@ -382,3 +382,15 @@ def __init__(self, *args, **kwargs):
               )
 
         )
+
+class CommentForm(ModelForm):
+    class Meta:
+        model = Comment
+        fields = ['comment']
+
+    def __init__(self, *args, **kwargs):
+        self.request = kwargs.pop('request', None)
+        super(CommentForm, self).__init__(*args, **kwargs)
+        self.helper = FormHelper(self)
+        self.helper.form_method = 'post'
```

Comment:
Need to add the following line:
self.helper.form_action = `whaterver your ReviewComment view's url is`

Reply:
I thought the `self.helper.add_input` took care of the form action?

Reply:
Ok I get it now, ignore above comment